api/v1alpha1/storageconsumer_types.go

```diff
@@ -40,6 +40,9 @@ const (
 type StorageConsumerSpec struct {
 	// Enable flag ignores a reconcile if set to false
 	Enable bool `json:"enable,omitempty"`
+	// StorageQuotaInGi describes quota for the consumer
+	// +optional
+	StorageQuotaInGi int `json:"storageQuotaInGi,omitempty"`
```

Comment:
The term is GigaBytes (Gib)
```suggestion
	StorageQuotaInGiB int `json:"storageQuotaInGiB,omitempty"`
```